modules/internal/String.chpl

```diff
@@ -631,6 +803,18 @@ module String {
       }
     }
 
+    /*
+     Checks if all the characters in the string are either uppercase (A-Z) or
+     uncased (not a letter).
+
+      :returns: * `true`  -- when there are no lowercase characters in the string.
+                * `false` -- otherwise
+     */
+    // TODO/BUG: these are implemented as documented above, but this is slightly
```

Comment:
I'd be more specific instead of saying "these", in case the functions get separated somehow.